l10n_jp_hr_employee/models/hr_qualification.py

```diff
@@ -15,14 +15,36 @@ class HrQualification(models.Model):
     _name = "hr.qualification"
     _order = "date_obtained"
 
-    name = fields.Char(required=True, help="e.g. CISA",)
+    name = fields.Many2one("hr.qualification.type", required=True)
     private_info_id = fields.Many2one("hr.private.info", string="Private Info",)
     employee_id = fields.Many2one(related="private_info_id.employee_id", store=True,)
     date_obtained = fields.Char("Date Obtained", required=True,)
     date_expiry = fields.Date("Valid Until",)
     reference = fields.Char()
     qualification_file = fields.Binary("Attachment",)
     qualification_file_filename = fields.Char("Attachment File Name",)
+    is_allowed_score = fields.Boolean(related="name.is_allowed_score", store=True)
+    score = fields.Char()
```

Comment:
`score` makes the scope of the field too narrow. Let's allow more flexibility.

```suggestion
    qualification_type_id = fields.Many2one("hr.qualification.type", required=True, string="Qualification Type")
    needs_description = fields.Boolean(related="qualification_type_id.needs_description")
    description = fields.Char()
```
No need to store `needs_description`.